configure

```diff
@@ -102,6 +102,8 @@ with_fuzzers=0
 floatabi=
 native=0
 forcesse2=0
+avx512flag="-mavx512f -mavx512dq -mavx512bw -mavx512vl"
```

Comment:
We should have a consistent order of the code. Either AVX512 first or AVX512VNNI first.

Reply:
Preferably oldest (in order when compilers implemented support) or "smallest" instruction set first, so when new ones are required, we don't need to insert them in the middle. This also allows easily leaving out any targets that some supported compilers don't support.

Reply:
Yeah it looks like `check_avx512_intrinsics` comes before `check_avx512vnni_intrinsics` so at least it is proper in CMake, but in configure it is in reverse.

Reply:
Flipping now